x-pack/plugins/cases/server/client/metrics/actions/actions.test.ts

```diff
@@ -25,6 +25,8 @@ const clientArgs = {
   authorization: { getAuthorizationFilter },
 } as unknown as CasesClientArgs;
 
+const constructorOptions = { caseId: '', casesClient: clientMock, clientArgs };
```

Comment:
nit: I think putting an empty string makes it a bit unclear if we are testing for "empty" values or not. If it is irrelevant to the tests `test-id` seems more appropiate.